xendit-java-lib/src/main/java/com/xendit/model/DisbursementClient.java

```diff
@@ -130,25 +139,89 @@ public AvailableBank[] getAvailableBanks(Map<String, String> headers) throws Xen
         RequestResource.Method.GET, url, headers, null, opt.getApiKey(), AvailableBank[].class);
   }
 
-  public Disbursement[] getByExternalId(String externalId) throws XenditException {
+  public DisbursementIDR[] getByExternalId(String externalId) throws XenditException {
     return getByExternalId(new HashMap<>(), externalId);
   }
 
-  public Disbursement[] getByExternalId(Map<String, String> headers, String externalId)
+  public DisbursementIDR[] getByExternalId(Map<String, String> headers, String externalId)
       throws XenditException {
     String url =
         String.format("%s%s%s", opt.getXenditURL(), "/disbursements?external_id=", externalId);
     return this.requestClient.request(
-        RequestResource.Method.GET, url, headers, null, opt.getApiKey(), Disbursement[].class);
+        RequestResource.Method.GET, url, headers, null, opt.getApiKey(), DisbursementIDR[].class);
   }
 
-  public Disbursement getById(String id) throws XenditException {
+  public DisbursementPH[] getByReferenceId(Map<String, String> headers, String referenceId)
+      throws XenditException {
+    String url =
+        String.format("%s%s%s", opt.getXenditURL(), "/disbursements?reference_id=", referenceId);
+    return this.requestClient.request(
+        RequestResource.Method.GET, url, headers, null, opt.getApiKey(), DisbursementPH[].class);
+  }
+
+  public DisbursementIDR getById(String id) throws XenditException {
     return getById(new HashMap<>(), id);
   }
 
-  public Disbursement getById(Map<String, String> headers, String id) throws XenditException {
+  public DisbursementIDR getById(Map<String, String> headers, String id) throws XenditException {
```

Comment:
`getIDRById`

help to do this kind of renaming to all, depending if they are using disbursement idr or ph.

Reply:
f8ab554fe61f23985a86e904b725d0a6fe209d99